frontend/app/src/main/java/com/quickpick/SwipeActivity.java

```diff
@@ -0,0 +1,26 @@
+package com.quickpick;
```

Comment:
Prefer to remove dead code if they're unused (ex the buttons, registerButtons, setOnClickListeners)